fe/fe-core/src/main/java/com/starrocks/catalog/CatalogRecycleBin.java

```diff
@@ -308,6 +308,11 @@ protected synchronized void eraseDatabase(long currentTimeMs) {
             RecycleDatabaseInfo dbInfo = entry.getValue();
             Database db = dbInfo.getDb();
             if (canErase(db.getId(), currentTimeMs)) {
+                List<Table> tables = db.getTables();
```

Comment:
There is no need to delete the table's incrementId for erase db. Because the db's table has been put to idToTableInfo, it will be erased on the eraseTable function.